scripts/repl.js

```diff
@@ -4,11 +4,11 @@
 const enchannelZMQ = require('../src');
 const createShellSubject = enchannelZMQ.createShellSubject;
 const createIOPubSubject = enchannelZMQ.createIOPubSubject;
-const uuid = require('uuid');
+const uuid = require('uuid/v4');
 const path = require('path');
 
 const chalk = require('chalk');
-const identity = uuid.v4();
+const identity = uuid();
```

Comment:
How would you do this with ES6 imports?
Right now `import v4 from 'uuid/v4';` works or should I use `require`?